lib/attrsets.nix

```diff
@@ -5,11 +5,11 @@ let
   inherit (builtins) head tail length;
   inherit (lib.trivial) and;
   inherit (lib.strings) concatStringsSep sanitizeDerivationName;
-  inherit (lib.lists) fold foldr concatMap concatLists;
+  inherit (lib.lists) foldr foldl' concatMap concatLists;
 in
 
 rec {
-  inherit (builtins) attrNames listToAttrs hasAttr isAttrs getAttr;
+  inherit (builtins) attrNames listToAttrs hasAttr isAttrs getAttr elemAt;
```

Comment:
This change creates `lib.attrsets.elemAt`, which isn't right. Could you move it to the let bindings above?

Reply:
sorry about this, I was too nervous I forgot to 1) check in a repl, 2) double check my code.

Reply:
Don't worry about it :) happens to me too sometimes.